applications/RANSApplication/custom_elements/convection_diffusion_reaction_residual_based_flux_corrected_element.cpp

```diff
@@ -276,32 +269,30 @@ double ConvectionDiffusionReactionResidualBasedFluxCorrectedElement<TDim, TNumNo
 
     r_current_data.CalculateConstants(rCurrentProcessInfo);
 
+    BoundedVector<double, TNumNodes> velocity_convective_terms;
+
     double scalar_multiplier = 0.0;
     for (IndexType g = 0; g < num_gauss_points; ++g) {
```

Comment:
another gigantic loop

Reply:
Same as the previous comment, It would be easier to follow the construction of local element matrices in one place rather than looking around to understand the formulations.